src/MainUtils.cpp

```diff
@@ -384,6 +384,7 @@ void addONNXToMLIRPasses(mlir::PassManager &pm) {
 
 void addONNXToKrnlPasses(mlir::PassManager &pm) {
   pm.addPass(mlir::createLowerToKrnlPass());
+  pm.addPass(mlir::createCanonicalizerPass());
```

Comment:
Can you add a comment indicating what the goal of this pass is, thanks

Reply:
Not needed anymore actually. Good catch.